src/components/history/History.tsx

```diff
@@ -96,7 +96,7 @@ export default function History(props: HistoryProps) {
         accessor: 'sectionNumber'
       },
       {
-        Header: 'Slot Id',
+        Header: 'Address',
         accessor: 'slotId'
```

Comment:
For 'Address' column , why the accessor is given as 'slotId' instead of address?